script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -29,74 +29,75 @@ Perhaps it's just me, but her bow doesn't seem as sharp as usual.
 <Page9>
 "...I wonder if she's worried about me. However, this kind of thing really does happen all the time."
 <Page10>
-I appreciate the concern, but it's nothing to be fussed about.
-If there is anything to be concerned about,
+I appreciate the concern, but it's nothing to be worried about.
+If there's anything to be concerned about,
```

Comment:
is / would be mismatch on these two lines
> If there would be anything to be concerned about,
it'd definitely be a serious wound like that